dist/backdraft.js

```diff
@@ -1275,8 +1275,18 @@ _.extend(Plugin.factory, {
 
     render : function() {
       var cells = this.findCells(), node;
-      _.each(this.columnsConfig, function(config) {
-        node = cells.filter(config.nodeMatcher(config));
+      var cleanConfigs = this.columnsConfig.map(function(currentValue,index){
```

Comment:
cleanConfigs is the array of column configs without any non-visible or undefined configs. I can't come up with a better name, `definedConfigs`, `availableConfigs` ?